Makefile

```diff
@@ -88,8 +88,11 @@ arch: build
 	cd integration-tests/ && ./test.sh arch amd64
 
 
-test-all-32: lint test test-int-32
-test-all: lint test test-int
+# TODO(stefan): test-int* are disabled, because dnstest.io is down and the integration tests fail.
+# test-all-32: lint test test-int-32
```

Comment:
Rather than disabling all of them, could we only remove the specific ones that fail?

Reply:
I've reenabled integration tests but only for centos, because the other platforms are broken or outdated.